conf/base.config

```diff
@@ -14,25 +14,123 @@ process {
   container = params.container
 
   cpus = { check_max( 1 * task.attempt, 'cpus' ) }
-  memory = { check_max( 8.GB * task.attempt, 'memory' ) }
+  memory = { check_max( 7.GB * task.attempt, 'memory' ) }
   time = { check_max( 2.h * task.attempt, 'time' ) }
 
   errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'finish' }
-  maxRetries = 1
+  maxRetries = 2
   maxErrors = '-1'
 
   // Process-specific resource requirements
-  withName: fastqc {
-    errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'ignore' }
+  withName:makeBWAindex {
+    cpus = { check_max( 12, 'cpus' ) }
+    memory = { check_max( 84.GB * task.attempt, 'memory' ) }
+    time = { check_max( 6.h * task.attempt, 'time' ) }
   }
-  withName: multiqc {
-    errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'ignore' }
+  withName:fastqc {
+    cpus = { check_max (6, 'cpus')}
```

Comment:
I see around three different kinds of machines you request here: 

12/84/6h
6/42/?

Maybe you could label your processes with something more generic like:

```
label "highmem"
```

and then have a single 

```
withLabel:highmem {
    memory = { check_max(84.GB * task.attempt, 'memory') }
}

Reply:
I did do this for the old ATACSeq pipeline but I saw that the `nf-core/rnaseq` pipeline was doing this explicitly for processes that required custom resources. Some processes will just require some more time. I think the process benchmarking we discussed will be very useful for this. Happy to change it though.

Reply:
I agree - maybe wait for the benchmarking and then adapt if required?

Reply:
Cool. Ill have another look over the requirements Ive set to try and simplify things in terms of the requirements.

The memory is basically 7 * cpu because we have 7G per CPU available at the Crick.

Reply:
This is now fixed with latest updates.